changelog.d/5318.misc

```diff
@@ -0,0 +1 @@
+Log the `since` token used and `next_batch` token returned when doing an incremental sync.
```

Comment:
Only updating the other file is enough, but do not worry I will delete this one later.

Reply:
Ah, I figured I may as well combine the two PRs into a changelog entry (as they really should have been one PR), but YMMV :)

Reply:
(FTR towncrier merges multiple changelog files with the same content into a single line in the final changelog)